doc/backing-store-crd.md

```diff
@@ -16,172 +22,206 @@ Adding support for a new type of backing-store is rather easy as it requires jus
-      namespace: noobaa
-    targetBucket: BUCKET
+      name: <>
+      namespace: <>
+    targetBucket: <>
   type: google-cloud-storage
 ```
 
-#### AZURE-BLOB type
-
-Create a cloud resource within the NooBaa brain and use BLOB API for storing encrypted chunks of data in Azure cloud.
+## Azure Blob Storage
+Uses the Azure Blob API for storing encrypted chunks of data in an Azure container
 ```shell
-noobaa -n noobaa backingstore create azure-blob bs --account-key KEY --account-name NAME --target-blob-container CONTAINER
+noobaa -n noobaa backingstore create azure-blob <BACKINGSTORE NAME> --account-key <> --account-name <> --target-blob-container <>
 ```
 ```yaml
 apiVersion: noobaa.io/v1alpha1
 kind: BackingStore
 metadata:
   finalizers:
   - noobaa.io/finalizer
-  labels:
-    app: noobaa
-  name: bs
-  namespace: noobaa
+  name: <>
+  namespace: <>
 spec:
   azureBlob:
     secret:
-      name: backing-store-azure-blob-bs
-      namespace: noobaa
-    targetBlobContainer: CONTAINER
+      name: <>
+      namespace: <>
+    targetBlobContainer: <>
   type: azure-blob
 ```
 
-#### PV-POOL type
+## Persistent Volume Pool
+Creataes a StatefulSet with a PVC mounted in each pod. Each resource will connect to the NooBaa core and provide the PV filesystem storage to be used for storing encrypted chunks of data. It is possible to configure the number of pods to be used and their PV size.
+
+Each PV needs at least 17 Gibibytes to function properly. The bare minimum defined in the source code is 16 Gibibytes, but it is better to take some extra space.
 
-Create NooBaa resources StatefulSet with PVC mounted in each pod. Each resource will connect to the NooBaa core and provide the PV filesystem storage to be used for storing encrypted chunks of data. It is possible to configure the number of pods to be used and their PV size.
+*In the YAML, the value of `storage` should adhere to the [K8s Memory Resource Units format](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory).
 ```shell
-noobaa -n noobaa backingstore create pv-pool bs --num-volumes 3 --pv-size-gb 32 --storage-class STORAGE-CLASS-NAME
+noobaa -n noobaa backingstore create pv-pool <BACKINGSTORE NAME> --num-volumes <> --pv-size-gb <INT >= 17> --storage-class <THE SC THAT WILL BE USED FOR PV PROVISIONING>
 ```
 ```yaml
 apiVersion: noobaa.io/v1alpha1
 kind: BackingStore
 metadata:
   finalizers:
   - noobaa.io/finalizer
-  labels:
-    app: noobaa
-  name: bs
-  namespace: noobaa
+  name: <>
+  namespace: <>
 spec:
   pvPool:
-    numVolumes: 3
+    numVolumes: <>
     resources:
       requests:
-        storage: 32Gi
+        storage: <SEE NOTE ABOVE*>
     storageClass: STORAGE-CLASS-NAME
   type: pv-pool
 ```
 
+## Examples
+AWS S3
+```shell
+noobaa -n app-namespace backingstore create aws-s3 aws-backingstore --access-key AKIAIOSFODNN7EXAMPLE --secret-key wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY --target-bucket personal-bucket
```

Comment:
I would add a disclaimer that users should omit flags `--access-key` and `--secret-key` (as appears in the CLI):
The best practice is to omit those flags, in that case, the CLI will prompt to prompt and read it securely from the terminal to avoid leaking secrets in the shell history.